benchmarks/src/benchmark.cpp

```diff
@@ -2069,7 +2069,7 @@ void Benchmark::run_convert_utf32_to_utf16_llvm(size_t iterations) {
 void Benchmark::run_convert_utf8_to_utf16_utfcpp(size_t iterations) {
     const char*  data = reinterpret_cast<const char*>(input_data.data());
     const size_t size = input_data.size();
-    std::string_view data_string = std::string_view(data,size);
+    std::string_view data_string = std::string_view(data);
```

Comment:
Why did you change `std::string_view(data,size);` into `std::string_view(data);` ?

Reply:
The C++17 and C++11 interfaces caused errors with MSVC compiler:
```
D:\a\simdutf\simdutf\benchmarks\src\benchmark.cpp(2077,30): error C2672: 'utf8::utf8to16': no matching overloaded function found [D:\a\simdutf\simdutf\build\benchmarks\src\simdutf_benchmarks_benchmark.vcxproj]
  D:\a\simdutf\simdutf\benchmarks\src\benchmark.cpp(2110,30): error C2672: 'utf8::utf16to8': no matching overloaded function found [D:\a\simdutf\simdutf\build\benchmarks\src\simdutf_benchmarks_benchmark.vcxproj]
  D:\a\simdutf\simdutf\benchmarks\src\benchmark.cpp(2131,29): error C2672: 'utf8::utf8to32': no matching overloaded function found [D:\a\simdutf\simdutf\build\benchmarks\src\simdutf_benchmarks_benchmark.vcxproj]
  D:\a\simdutf\simdutf\benchmarks\src\benchmark.cpp(2159,29): error C2672: 'utf8::utf32to8': no matching overloaded function found [D:\a\simdutf\simdutf\build\benchmarks\src\simdutf_benchmarks_benchmark.vcxproj]
  ```
  
Could not determine the reason for this, though it seems to occur elsewhere, https://github.com/bfgroup/duck_invoke/issues/1
  
Therefore removed these interfaces and used  interface described at https://github.com/nemtrif/utfcpp#utf8utf16to8-2

The error does not occur in the original utfcpp repository, but some other error does occur. Trying to get a more complete test suite added https://github.com/nemtrif/utfcpp/pull/98

Reply:
I understand, but I refer to the string_view constructor specifically. Your code might be correct, but I am just curious.

Reply:
Looked up different ways to call this to see if it might resolve the problem that `no matching overloaded function could be found`.

Reply:
My understanding is that the size is not a direct input for u16string_view or u32string_view:
https://github.com/cplusplus/draft/blob/main/source/strings.tex#L564
https://www.learncpp.com/cpp-tutorial/introduction-to-stdstring_view/ 

string_view does seem to have advantages, but if it is not portable for all C++17 compilers, then the preference is to use the portable version of the code. Can check again with size present in case I missed something.